app/Http/Controllers/BillController.php

```diff
@@ -42,23 +44,33 @@ public function create()
     public function store(BillRequest $request)
```

Comment:
please investigate about transaction in Laravel and thinking about way to improve performance for this.